geometric_features/geometric_features.py

```diff
@@ -80,7 +80,7 @@ def read(self, componentName, objectType, featureNames=None, tags=None,
 
         Parameters
         ----------
-        componentName : {'bedmachine', 'bedmap2', 'iceshelves', 'landice', 'natural_earth', 'ocean'}
+        componentName : {'bedmachine', 'bedmap2', 'iceshelves', 'landice', 'natural_earth', 'ocean','seaice'}
```

Comment:
```suggestion
        componentName : {'bedmachine', 'bedmap2', 'iceshelves', 'landice', 'natural_earth', 'ocean', 'seaice'}
```
(check line length - might be too long)